src/index.js

```diff
@@ -2,6 +2,7 @@ import _ from 'lodash';
 import './style.css';
 import refreshImage from './refresh.png';
 import * as functionality from './modules/addRemove.js';
+import * as interactions from './modules/interaction.js';
```

Comment:
- [OPTIONAL ] These imports are not used in this file because you are not exporting anything from the `addRemove` & `interactions`. Please make sure you are exploring the methods / class definition in those files 🤔